pkg/config/render_config.go

```diff
@@ -0,0 +1,84 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"html/template"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// context contains the context used to render the config file template
+type context struct {
+	Common            bool
+	Agent             bool
+	Metadata          bool
+	Dogstatsd         bool
+	JMX               bool
+	Autoconfig        bool
+	Logging           bool
+	Autodiscovery     bool
+	DockerTagging     bool
+	KubernetesTagging bool
+	ECS               bool
+}
+
+func mkContext(buildType string) context {
+	buildType = strings.ToLower(buildType)
+
+	switch buildType {
+	case "agent":
+		return context{
+			Common:            true,
+			Agent:             true,
+			Metadata:          true,
+			Dogstatsd:         true,
+			JMX:               true,
+			Autoconfig:        true,
+			Logging:           true,
+			Autodiscovery:     true,
+			DockerTagging:     true,
+			KubernetesTagging: true,
+			ECS:               true,
+		}
+	case "dogstatsd":
+		return context{
+			Common:        true,
+			Dogstatsd:     true,
+			DockerTagging: true,
```

Comment:
we should also activate the logging part.

Reply:
done

Reply:
DSD's tagger also uses KubernetesTagging and ECS